extension/scripts/features/highlightprops/ttHighlightProps.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+(async () => {
+	const feature = featureManager.registerFeature(
+		"Highlight Property",
+		"sidebar",
+		() => settings.pages.sidebar.upkeepPropHighlight,
+		null,
+		addHighlight,
+		removeHighlight,
+		{
+			storage: ["settings.pages.sidebar.upkeepPropHighlight"],
+		},
+		async () => {
+			if (!hasAPIData()) return "No API access.";
+			await requireSidebar();
```

Comment:
These should be in the execute function (addHighlight), unless it's really needed in the requirement function.

Reply:
It should have a 'mobile' check though.